Cargo.toml

```diff
@@ -3,3 +3,11 @@
 
 [workspace]
 members = ["rust"]
+
+[profile.release]
+panic = "abort"
+codegen-units = 1
+lto = true
+
+## needs cargo >= 1.59 (https://doc.rust-lang.org/cargo/reference/unstable.html#profile-strip-option)
+# strip = true
```

Comment:
This is not necessarily desirable; if something goes wrong then having the debug symbols available would massively help.

Maybe we want [`split-debuginfo`](https://doc.rust-lang.org/cargo/reference/profiles.html#split-debuginfo) or something here, but I think that needs a much later Rust compiler from memory.

Reply:
why would you ship debug info for releases?
if you target a debug release nothing will change

Reply:
Mostly wondering if we would ever want to use `gdb` in production or on a core dump if something goes horribly wrong. Hard for me to say, will accept what others think